src/fonts/fonts.css

```diff
@@ -1,9 +1,27 @@
+/* ibm-plex-sans-700 - latin-ext_latin */
 @font-face {
-    font-family: IBM Plex Sans;
-    src: url(./IBMPlexSans-Bold.ttf);
+    font-family: 'IBM Plex Sans';
+    font-style: normal;
+    font-weight: 700;
+    src: url('./IBMPlexSans-Bold/IBMPlexSans-Bold.eot'); /* IE9 Compat Modes */
+    src: local('IBM Plex Sans Bold'), local('IBMPlexSans-Bold'),
+         url('./IBMPlexSans-Bold/IBMPlexSans-Bold.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+         url('./IBMPlexSans-Bold/IBMPlexSans-Bold.woff2') format('woff2'), /* Super Modern Browsers */
+         url('./IBMPlexSans-Bold/IBMPlexSans-Bold.woff') format('woff'), /* Modern Browsers */
```

Comment:
woff is supported by everything apart from IE 8 (pretty sure our app does not work there either way) and opera mini